R/FilteredData.R

```diff
@@ -726,6 +720,11 @@ FilteredData <- R6::R6Class( # nolint
                 }
               )
             )
+          ),
+          div(
+            id = ns("filters_active_count"),
+            style = "display: none;",
+            textOutput(ns("teal_filters_count"))
```

Comment:
```suggestion
          shinyjs::hidden(div(
            id = ns("filters_active_count"),
            textOutput(ns("teal_filters_count"))
```

Reply:
reason for this is that `shinyjs::show/hide` has it's own way of handling this. At this moment they also append `style = "display:none` but also change the class of the element. If anything in the future change with `shinyjs` our `display: "none"` might affect how `shinyjs` works.